supervisor/pip.sls

```diff
@@ -51,6 +54,15 @@ supervisor_init:
     - user: root
     - group: root
     - mode: 744
+  module.run:
```

Comment:
If we do that, then we need to add this same conditional around line 21 where we 'watch' this state. I like it better just having the conditional in one place. I tried to have the `supervisor_service` state just watch the `file: supervisor_init` but that wasn't sufficient.